flybywiresim/a32nx
# fix(lights): Fix broken daytime lights

Fixes #5292 

## Summary of Changes


1. Fixed lighting issue related to day/night-cycle by explicitly setting valid color value for emitter.
2. Increased pedestal ambient light slightly

## Testing instructions
1. Spawn anywhere
2. Power aircraft
3. Turn on `MAIN PNL FLOOD LT` and `FLOOD LT PED`
4. Use the Time/Weather dialog to change from day to night
5. Verify that lights not turns off during daytime

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
